app/javascript/components/student/mentoring-session/mentoring-request/MentoringRequestFormComponents/TrackObjectivesTextArea.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import type { MentorSessionTrack } from '@/components/types'
+
+export const TrackObjectivesTextArea = React.forwardRef<
```

Comment:
Any reason why you didn't go for:

```suggestion
export const TrackObjectives = React.forwardRef<
```

That way the component name is not tied to the implementation.

This same comment applies to the other component.

Reply:
I agree with this, but it is harder to see in the markup that this is a textarea. that's the only reason I chose this name.

Reply:
Do you mean in the generated HTML or at the calling site? Is it important to know that it is a textarea?